diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter2/kdoc/KdocComments.kt

```diff
@@ -209,17 +214,21 @@ class KdocComments(configRules: List<RulesConfig>) : DiktatRule(
         // if property is documented with KDoc, which has a property tag inside, then it can contain some additional more complicated
         // structure, that will be hard to move automatically
         val isFixable = propertyInLocalKdoc == null
-        KDOC_NO_CONSTRUCTOR_PROPERTY.warnAndFix(configRules, emitWarn, isFixable, prevComment.text, prevComment.startOffset, node, isFixable) {
```

Comment:
There was a bug all this time, we were luckily only receiving `false` here, it seems. Or never running diktat in check mode

Reply:
oh

Reply:
Yes, I discovered this bug when my new tests have started failing. So now it's covered by those.